pfcpiface/parse-far.go

```diff
@@ -40,13 +42,20 @@ type far struct {
 	fseID   uint64
 	fseidIP uint32
 
-	dstIntf      uint8
-	applyAction  uint8
-	tunnelType   uint8
-	tunnelIP4Src uint32
-	tunnelIP4Dst uint32
-	tunnelTEID   uint32
-	tunnelPort   uint16
+	dstIntf         uint8
+	pfcpSMReqFlags  uint8
+	sendEndMarker   bool
+	applyAction     uint8
+	tunnelType      uint8
+	tunnelIP4Src    uint32
+	tunnelIP4Dst    uint32
+	tunnelTEID      uint32
+	tunnelPort      uint16
+	oldTunnelType   uint8
+	oldTunnelIP4Src uint32
+	oldTunnelIP4Dst uint32
+	oldTunnelTEID   uint32
+	oldTunnelPort   uint16
```

Comment:
Would we be able to get rid of extra state added if we craft the endmarker at PFCP layer instead of BESS layer?
At PFCP layer we know its an explicit update FAR message and we can call the interface method registered to send the end marker payload out.

Reply:
done